_data/schedule/saturday.yml

```diff
@@ -1,3 +1,7 @@
+- start: 1:00 AM
+  end: 3:00 AM
+  show: the-topdollar-hour
+
```

Comment:
Above, I add the show to the schedule on the correct day. All times are EST (for now).